### PR TITLE
chore: use near-abi-rs

### PR DESCRIPTION
**Pull request by itegulov**

### CODE REVIEW

File: examples/delegator-generation/Cargo.toml [hunk 1]
```diff
@@ -7,16 +7,14 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = { version = "4.1.0-pre.0", features = ["abi"] }
+near-sdk = { version = "4.1.0-pre.1", features = ["abi"] }
```
Comment (miraclx) [inline examples/delegator-generation/Cargo.toml]:
```suggestion
near-sdk = { version = "4.1.0-pre.2", features = ["abi"] }
```

File: near-sdk-abi/src/lib.rs [hunk 2]
```diff
@@ -98,6 +98,7 @@ impl Generator {
                 mod_name.map(|n| format_ident!("{}", n)),
             );
             let token_stream = quote! {
+                #![allow(unused_imports)]
                 use serde::{Deserialize, Serialize};
                 #token_stream
```
Comment (miraclx) [inline near-sdk-abi/src/lib.rs:103]: Are these imports used? Can't seem to find where.

**Review by miraclx — APPROVED** [review]

File: near-sdk-abi/src/lib.rs (hunk 2 above)
Comment (itegulov) [reply near-sdk-abi/src/lib.rs:103]: Yeah unfortunately schemafy generates code that uses unqualified serde derivation. Here for example: https://github.com/Marwes/schemafy/blob/master/schemafy_lib/src/lib.rs#L560.

The small example that we have does not have this issue, but it appears when you have enums or proper structs.

Comment (miraclx) [reply near-sdk-abi/src/lib.rs:103]: Ah, makes sense.

**Pull request merged**